albam/engines/mtfw/texture.py

```diff
@@ -57,6 +57,19 @@ class TextureType(Enum):  # TODO: TextureTypeSlot
     SPECULAR_BLEND = 22
 
 
+SYMBOL_TEX_TYPES = {
+    1: "basemap",
+    2: "normalmap",
+    3: "maskmap",
+    4: "lightmap",
+    5: "shadowmap",
+    6: "additionalmap",
```

Comment:
what is additional map? Is it original?

Reply:
The additional map is _AM, a black and white mask used in some stages for transparency